README.md

```diff
@@ -61,3 +72,7 @@ You need to create an account in Trello & obtained a TRELLO_API_KEY & TRELLO_API
 - SIGN UP https://trello.com/signup
 
 - AUTH https://developer.atlassian.com/cloud/trello/guides/rest-api/api-introduction/#managing-your-api-key
+
+
+## Testing 
+You need to add pytest to poetry as this will be 
```

Comment:
It feels like this has finished abruptly, but actually it's worth noting that future developers _don't_ need to add pytest, since it will automatically be installed when they run `poetry install`